kahuna/public/js/search/results.html

```diff
@@ -13,14 +13,43 @@
 
     <div ng:if="ctrl.totalResults > 0">
 
-        <div class="image-results-count">
-            {{ctrl.totalResults | toLocaleString}} matches
-            <span ng:if="ctrl.inSelectionMode()">({{ctrl.selectedImages.size}} selected)</span>
-            <button ng:show="ctrl.newImagesCount > 0"
-                    ng:click="ctrl.revealNewImages()"
-                    class="image-results-count__new">
-                {{ctrl.newImagesCount | toLocaleString}} new
-            </button>
+        <div class="results-toolbar">
+            <div class="image-results-count">
+                {{ctrl.totalResults | toLocaleString}} matches
+                <span ng:if="ctrl.inSelectionMode()">({{ctrl.selectedImages.size}} selected)</span>
+                <button ng:show="ctrl.newImagesCount > 0"
+                        ng:click="ctrl.revealNewImages()"
+                        class="image-results-count__new">
+                    {{ctrl.newImagesCount | toLocaleString}} new
+                </button>
+            </div>
+            <div class="sort-direction">
+                <div class="radio-list">
+                    <div class="radio-list__item">
+                        <input type="radio"
+                               id="sort-direction__desc"
+                               class="radio-list__circle"
+                               name="sort-direction"
+                               value="DESC"
+                               ng:model="ctrl.filter.sortDirection">
+                        <label for="sort-direction__desc" class="radio-list__label">
+                            Most Recent First
```

Comment:
@theefer Yep, that also works (and is shorter!). Sentence case whatever it happens to be!